reference/orders.v2.oas2.yml

```diff
@@ -3490,6 +3493,10 @@ components:
           example: 52
           description: ID of the associated gift certificate.
           nullable: true
+        discounted_total_inc_tax:
+          type: string
+          example: '0.0000'
+          description: The discounted line item total.
```

Comment:
```suggestion
          description: |-
             Represent the correct total amount of the line item after deducting all the discounts and including the tax. This number can be used for accounting purpose.
       
             Why we cannot use the `total_inc_tax` amount? It's because `total_ex_tax + total_tax != total_inc_tax` in multiple scenarios, especially when we have a mix-match of entered price exclusive of tax, and display price inclusive of tax, and vice versa. So that field `total_inc_tax` is not reliable for accounting purpose and the value may be wrong.
```